ui/test/nuxeo-collapsible.test.js

```diff
@@ -0,0 +1,160 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import '../widgets/nuxeo-collapsible.js';
+import { html, fixture, flush, isElementVisible, waitChanged } from '../../testing-helpers';
+import '../nuxeo-icons';
+/* eslint-disable no-unused-expressions */
+
+suite('nuxeo-collapsible', () => {
+  suite('Collapse behaviour', () => {
+    test('Should display nuxeo-collapsible when declaring the element', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible></nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      expect(isElementVisible(nuxeoCollapsible.shadowRoot.querySelector('button'))).to.be.true;
+    });
+
+    test('Should show and hide content when toggle is activated', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible>
+            <span slot="heading">Content</span>
+            <div>Collapsed content</div>
+          </nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      const content = nuxeoCollapsible.shadowRoot.querySelectorAll('slot')[2].assignedNodes()[2];
+      const ironCollapse = nuxeoCollapsible.shadowRoot.querySelector('iron-collapse');
+      if (isElementVisible(content)) {
+        await waitChanged(ironCollapse, 'aria-hidden');
+      }
+      expect(isElementVisible(content)).to.be.false;
+
+      nuxeoCollapsible.expanded = true;
+      if (!isElementVisible(content)) {
+        await waitChanged(ironCollapse, 'aria-hidden');
+      }
```

Comment:
There is no other value we can trust? According to this test it has `aria-hidden` when it is showing the content but also when it is hiding 🤔